SPECS/cert-manager/cert-manager.spec

```diff
@@ -67,11 +61,14 @@ Webhook component providing API validation, mutation and conversion functionalit
 %setup -q -T -D -a 1
 
 %build
-go build -o bin/acmesolver cmd/acmesolver/main.go
-go build -o bin/cainjector cmd/cainjector/main.go
-go build -o bin/controller cmd/controller/main.go
-go build -o bin/cmctl cmd/ctl/main.go
-go build -o bin/webhook cmd/webhook/main.go
+
+LOCAL_BIN_DIR=$(realpath bin)
+# LOCAL_MOD_DIR=$(realpath vendor)
```

Comment:
Why the commented line? A leftover after testing?

Reply:
yes that's exactly why; good catch